src/hooks/validators/getError.ts

```diff
@@ -0,0 +1,53 @@
+import { FieldError } from 'react-hook-form'
+
+import { ErrorTypes, ErrorMessage } from './types'
+import { JSONSchemaType } from '../../JSONSchema'
+
+export const getError = (
+  errors: FieldError | undefined,
+  currentObject: JSONSchemaType,
+  isRequired: boolean,
+  minimum?: number,
+  maximum?: number,
+  step?: number | 'any'
+) => {
+  if (errors) {
```

Comment:
Treat edge case as edge

```jsx
if (!errors) {
  return undefined
}
```

Reply:
Yeah, I like to keep a `return as fast as possible` approach and it prevents another level of indentation for the rest of the code.